modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java

```diff
@@ -77,12 +77,21 @@ public int doStartTag() throws JspException {
 				(CommerceContext)httpServletRequest.getAttribute(
 					CommerceWebKeys.COMMERCE_CONTEXT);
 
+			_commerceChannelId = 0;
+
+			if (commerceContext != null) {
+				_commerceChannelId = commerceContext.getCommerceChannelId();
+			}
+
+			if (_commerceChannelId == 0) {
+				return SKIP_BODY;
+			}
```

Comment:
The runtime will continue with no `commerceContext`.

Is this what we want? If not, a simple
```
if (commerceContext == null || commerceContext.getCommerceChannelId() == 0) { ... }
```
will do the job.